azurerm/resource_arm_servicebus_queue_test.go

```diff
@@ -551,3 +583,26 @@ resource "azurerm_servicebus_queue" "test" {
 }
 `, rInt, location, rInt, rInt)
 }
+
+func testAccAzureRMServiceBusQueue_maxDeliveryCount(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+    name = "acctestRG-%d"
```

Comment:
Could we align all the assignments in the TF code to match the other tests?

Reply:
Definitely. Sorry for not noticing the inconsistency.